### PR TITLE
Add a possibility to configure nodePort explicitly for OpenVPN

### PR DESCRIPTION
**Pull request by alexclear**
*****************************************************************************************

Thank you for contributing to kubernetes/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/kubernetes/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/kubernetes/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the 
history. This will make it easier to identify new changes. The PR will be squashed 
anyways when it is merged. Thanks.

*****************************************************************************************

### CODE REVIEW

File: stable/openvpn/templates/openvpn-service.yaml [hunk 1]
```diff
@@ -12,6 +12,9 @@ spec:
       port: {{ .Values.service.externalPort }}
       targetPort: {{ .Values.service.internalPort }}
       protocol: {{ .Values.openvpn.OVPN_PROTO | upper }}
+{{- if hasKey .Values.service "nodePort" }}
```
Comment (eicnix) [inline stable/openvpn/templates/openvpn-service.yaml]: IMO you should check if .Values.service.type equals NodePort. 

Can you add the nodePort parameter to the values.yaml and add documentation about this to the readme.md and the notes.txt?

**Comment by alexclear** [comment]
@eicnix Done

**Pull request merged**